fix: A value is trying to be set on a copy of a slice from a DataFrame

abhishek-amar opened the pull request:
- Fixes warning caused by setting a value on copy of a DataFrame by explicitly creating a copy.
- Link #5582

deeleeramone commented:
Thanks for the patch! This works, however, to merge your branch we need to rebase it to the `/develop` branch.  You should be able to change it with the "Edit" button.

abhishek-amar commented:
> Thanks for the patch! This works, however, to merge your branch we need to rebase it to the `/develop` branch. You should be able to change it with the "Edit" button.

Sorry for the confusion, I have changed the PR to merge with the `develop` branch and rebased it.

deeleeramone reviewed (approved):
Thanks for the PR!
